en/docs/setup/migrating-preparing-for-migration.md

```diff
@@ -21,6 +21,113 @@
 
 With WSO2 Identity Server 5.11.0, groups and roles are separated. For more information, see [What Has Changed in 5.11.0](../../setup/migrating-what-has-changed#group-and-role-separation).
 
+## Groups and Roles Improvements Migration
+
+With WSO2 Identity Server 5.12.0, groups and roles improvements are introduced. For more information, see [What Has Changed in 5.11.0](../../setup/migrating-what-has-changed#roup and role separation Improvements).
+
+Please note that the abbreviations mentioned in that section is also used in here.
+
+Please refer to the below sections related to this improvement.
+
+### Claim configuration changes
+
+#### Following changes are added to the product claims.
```

Comment:
why do we need this as a heading?

Reply:
Fixed with fa6da06b8ecc8fd6ac34122f47ef8e9050296214